plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -151,6 +151,13 @@ def api_command(module, command, name, args):
     return api.Command[command](name, **args)
 
 
+def api_command_no_args(module, command, options):
```

Comment:
Why is this called `api_command_no_args`? You renamed args to options from api_command, but `args` is still there somehow. Please change the name to something like `api_command_no_name` and keep the name `args`.

Reply:
Fixed.